### DISCUSSION TITLE
I have applied  student developer benefits

### DISCUSSION THREAD

**Discussion by RahulDeveloper07** [community/community#17729] [GitHub Education]
I have trying to upload students id but i doesn't work?
Please solve it.

**Comment by ghost** [comment]
Are you sure the id you are trying to upload is dated? If not, just use a fee receipt or an official transcript that might help. If it still doesn't work, try contacting support. Just remember that any doc you upload _must_ be dated.

**Comment by RahulDeveloper07** [comment]
No i am trying  to upload  any of documents  but ...its not working.
Camera  should be not open...

**Reply by EnisR** [reply]
When I applied for my student developer pack all it was needed was one of the documents that shows that you are a student at the given university/school, I uploaded a .jpg file of my info and student id that was up-to-date and they accepted it within 15 minutes. Hope this helps somehow. Again no need for picture of you but an up-to-date doc of your student ID,
